src/ResourceManager/SiteRecovery/Commands.SiteRecovery/Vault/SetAzureSiteRecoveryVaultSettings.cs

```diff
@@ -31,21 +31,21 @@ public class SetAzureSiteRecoveryVaultSettings : SiteRecoveryCmdletBase
         /// <summary>
         /// Gets or sets vault Object.
         /// </summary>
-        [Parameter(ParameterSetName = ASRParameterSets.Default, Mandatory = true, ValueFromPipeline = true)]
+        [Parameter(ParameterSetName = ASRParameterSets.Default, Mandatory = true)]
```

Comment:
We still want to retain this for customers who are not passing objects through the pipeline